include/deal.II/base/derivative_form.h

```diff
@@ -110,7 +110,13 @@ class DerivativeForm
   DerivativeForm<1, spacedim, dim, Number> transpose () const;
 
   /**
-   * Computes the volume element associated with the jacobian of the
+   * Compute the Frobenius norm of this form, i.e., the expression
+   * $\sqrt{\sum_{ij} |DF_{ij}|^2}$.
+   */
+  double norm () const;
```

Comment:
Shouldn't the return type be either `Number` or `numbers::NumberTraits<Number>::real_type`?